src/App.js

```diff
@@ -27,6 +29,9 @@ function App() {
               <Route path="/blog" component="#" />
               <Route path="/cart" component={Cart} />
               <Route path="/admin" component={AdminDashboard} />
+              <Route path="/cart" component="#" />
```

Comment:
why did you delete both of them.. we had a route for CouponsPage this shouldn't be deleted, and the route on line 32 should be deleted cause at line 30 we already have a route for cart

Reply:
I have commented on this route for my last review but you deleted other things and not the thing i commented on